src/components/dialogs/dataSources/CDC.svelte

```diff
@@ -3,16 +3,23 @@
   import { cdcLocations as regions } from '../../../data/data';
   import SelectField from '../inputs/SelectField.svelte';
   import TextField from '../inputs/TextField.svelte';
+  import { apiKey } from '../../../store';
 
   export let id: string;
 
   let locations = regions[0].value;
-  let auth = '';
+  let auth = $apiKey;
 
   export function importDataSet() {
     return importCDC({ locations, auth });
```

Comment:
is the `auth` variable still necessary?
```suggestion
  export function importDataSet() {
    return importCDC({ locations, apiKey });
```
(and similarly in the other data source dialogs)

Reply:
It's still necessary. `apiKey` itself is a `Writable<string>` store, and `let auth = $apiKey` acts as a sort of getter.

Reply:
really?  that is very strange to me.  what if you replaced `auth` with `$apiKey` instead of just `apiKey`?